src/hclib-runtime.c

```diff
@@ -354,14 +355,15 @@ static inline void check_out_finish(finish_t *finish) {
         // hc_atomic_dec returns true when finish->counter goes to zero
         if (hc_atomic_dec(&(finish->counter))) {
 #if HCLIB_LITECTX_STRATEGY
```

Comment:
I think we need to leave the alternatives. I have some other strategies that I will commit in the future, each with its own tradeoffs. However, I think having `HCLIB_LITECTX_STRATEGY` as the new default is a good thing.